File: src/test/kotlin/com/benggrea/springredisstrategy/web/MainTest.java
package com.benggrea.springredisstrategy.web;

import org.junit.jupiter.api.Test;

public class MainTest {

    @Test
    void test() {
        int[] arr = {1, 3, 4};
        arr[0] = arr[1] = 0;

        System.out.println(arr);

    }
}

File: src/test/kotlin/com/benggrea/springredisstrategy/web/MainTest.kt
package com.benggrea.springredisstrategy.web

import io.kotest.core.spec.style.AnnotationSpec
import io.kotest.matchers.shouldBe

class MainTest : AnnotationSpec() {

    @Test
    fun `테스트 샘플`() {
        val name = "kyunam"
        name shouldBe "kyunam"
    }
}
